roles/installer/tasks/resources_configuration.yml

```diff
@@ -13,9 +13,17 @@
       - status.phase=Running
   register: tower_pod
 
+- name: Set the resource pod as a variable.
```

Comment:
This will prevent us from grabbing a pod that is in terminating state.  It also ensures that only one pod is grabbed (the oldest).

Reply:
Just adding the corresponding commit message in case it got missed

> commit b3a74362aff871f4eb9b9ba73ea2b4ddf1ba46ff
> 
>     Make AWX Pod variable to be calculated respecting `creationTimestamp` and `deletionTimestamp`
> 
>     Do not consider Pods marked for deletion when calculating tower_pod to
>     address replicas scale down case - where normally Pods spawned recently
>     are being taken for removal. As well as the case when operator kicked
>     off but some old replicas are still terminating.
> 
>     Respect `creationTimestamp` so to make sure that the newest Pod is taken
>     after Deployment application, in which case multiple RS Pods (from old
>     RS and new RS) could be running simultaneously while the rollout is
>     happening.

I've also encountered a couple of cases when `Terminating` pod was picked up, and running `awx-manage` command on it during the later tasks was not possible (the pod was gone already, etc).

Here indeed the oldest pod is taken, while [later](https://github.com/ansible/awx-operator/pull/1222/files#diff-d8ddd3baf20d7042360b2b1c2c284a8545d24149814ce705535e71ec203a028cR275) in the code the newest (the one most recently created) is used.